File: app/src/main/java/com/app/greatfood/WelcomeActivity.java
package com.app.greatfood;

import android.content.Intent;
import androidx.appcompat.app.AppCompatActivity;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;

public class WelcomeActivity extends AppCompatActivity {

    Button next;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_welcome);
    }

    public void Ingresar (View view) {
        Intent ingresar = new Intent (this, MainActivity.class);
        startActivity(ingresar);
    }

    }

File: app/src/main/java/com/project/greatfood/activities/WelcomeActivity.kt
package com.project.greatfood.activities

import android.content.Intent
import android.os.Bundle
import android.view.View
import android.widget.Button
import androidx.appcompat.app.AppCompatActivity
import com.project.greatfood.R

class WelcomeActivity : AppCompatActivity() {
    var next: Button? = null
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_welcome)
    }

    fun Ingresar(view: View?) {
        val ingresar = Intent(this, MainActivity::class.java)
        startActivity(ingresar)
    }
}